images/win/toolsets/toolset-2019.json

```diff
@@ -121,5 +121,75 @@
                 "4.3.0"
             ]
         }
-    ]
+    ],
+    "android": {

```

Comment:
Let's put all these arrays in line
```
"platform_list": [
            "android-30", "android-29", "android-28", "android-27", "android-25", "android-24", "android-23", "android-26", "android-22", "android-21", "android-19"
],
```
Current approach takes too much lines and decrease readability

Reply:
Done